xml/art_sles_amd-sev.xml

```diff
@@ -0,0 +1,431 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE appendix
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+  <!ENTITY productnamex "<phrase xmlns='http://docbook.org/ns/docbook' os='sles;sled;osuse'>&productname;</phrase><phrase xmlns='http://docbook.org/ns/docbook' os='article'>&sls;</phrase>">
+  <!ENTITY productnumberx "<phrase xmlns='http://docbook.org/ns/docbook' os='sles;sled;osuse'>&productnumber;</phrase><phrase xmlns='http://docbook.org/ns/docbook' os='article'>15</phrase>">
+]>
+
+<!--
+  IMPORTANT: When editing this file, make sure to keep the article version
+  (art_amd-sev.xml) up-to-date.
+-->
+
+<appendix xml:id="app.amd-sev"
+ xmlns="http://docbook.org/ns/docbook" version="5.0"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>AMD Secure Encrypted Virtualization</title>
+ <info>
+  <abstract>
+   <para>
+    AMD's Secure Encrypted Virtualization (SEV) adds a facility to Linux's
+    built-in Kernel-based Virtual Machine (&kvm;) hypervisor, which allows the
+    memory assigned to running virtual machines to be encrypted for increased
+    security.
+   </para>
+  </abstract>
+ </info>
+
+ <para>
+  This document aims to provide a basic understanding of how SEV works, how to
+  enable and configure it, and some of the limitations and restrictions that its
+  use causes as compared to non-encrypted virtualization.
+ </para>
```

Comment:
The two paragraphs above should become the abstract of the article

Reply:
Done. I did not know that an abstract could contain more than a single paragraph.